scripts/build/entries/index.ts

```diff
@@ -1340,6 +1337,36 @@ export const buildEntries = {
       ],
     },
   },
+  'eslint-plugin-storybook': {
+    prebuild: async (cwd) => {
+      const ESLINT_PLUGIN_PREBUILD_SCRIPT_PATH = join(
+        DIR_CODE,
+        'lib',
+        'eslint-plugin',
+        'scripts',
+        'update-all.ts'
+      );
```

Comment:
we're passed `cwd` that should make this significantly shorter, and more robust, if we ever decide to move the code around to a different directory

Reply:
I'll take a look at this in the cleanup PR instead where this is moved anyways